src/util/value.ts

```diff
@@ -46,6 +46,37 @@ export type Rect = {
   north: number;
 };
 
+export type Plane = {
+  normal: {
+    x: number;
+    y: number;
+    z: number;
+  };
+  distance: number;
+};
+
+export type ClippingPlaneCollection = {
+  planes?: Plane[];
+  dimensions?: {
+    /**
+     * x-axis
+     */
+    width?: number;
+    /**
+     * y-axis
+     */
+    length?: number;
+    /**
+     * z-axis
+     */
+    height?: number;
+  };
+  heading?: number;
+  pitch?: number;
+  roll?: number;
+  height?: number;
+} & LatLng;
```

Comment:
```suggestion
// ideal for plugin developers, but it's hard to implement it with Cesium
export type Plane = {
  location: LatLngHeight;
  width: number;
  height: number;
  heading: number;
  pitch: number;
};

// Familiar with Cesium
export type EXPERIMENTAL_Clipping = {
  planes: { normal: [number, number, number]; distance: number }[];
  modelMatrix: number[];
}
```

Reply:
I fixed this comment. [fix: use experimental clipping](https://github.com/reearth/reearth-web/pull/338/commits/60d8d4b961f99da27b7f165688166bd729c2b7a2)